src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const partner = person.sex === 'f'
+    ? 'husband'
+    : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+      {person.age > 0
+        && <p className="Person__age">{`I am ${person.age}`}</p>
+      }
+
+      {person.isMarried === true
```

Comment:
```suggestion
      {person.isMarried
```
It will work the same